Dockerfile.openshift

```
@@ -1,12 +1,12 @@
 # This dockerfile is specific to building Multus for OpenShift
-FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.25-openshift-4.22 AS build
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.26-openshift-5.0 AS build
 
 # Add everything
 ADD . /usr/src/multus-networkpolicy
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)